src/test/java/htsjdk/samtools/cram/structure/CramCompressionRecordTest.java

```diff
@@ -12,19 +13,24 @@
  * Created by vadim on 28/09/2015.
  */
 public class CramCompressionRecordTest extends HtsjdkTest {
-    @Test
-    public void test_getAlignmentEnd() {
+    @DataProvider(name = "tf")
+    private Object[][] tf() {
+        return new Object[][] { {true}, {false}};
+    }
+
+    @Test(dataProvider = "tf")
+    public void test_getAlignmentEnd(final boolean APDelta) {
```

Comment:
Except when referring to the actual APDelta data series, it might be more useful (in the tests and in the code) to call these arguments `usePositionDeltaEncoding`, or something like that, in order to underscore what its actually referring to.